packages/jbrowse-plugin-apollo/src/index.ts

```diff
@@ -129,6 +134,22 @@ export default class ApolloPlugin extends Plugin {
           ])
         },
       })
+      pluginManager.rootModel.appendToMenu('Apollo', {
+        label: 'Delete Assembly',
+        onClick: (session: AbstractSessionModel) => {
+          session.queueDialog((doneCallback) => [
+            DeleteAssembly,
+            {
+              session,
+              handleClose: () => {
+                doneCallback()
+              },
+              changeManager: (session as ApolloSessionModel).apolloDataStore
+                .changeManager,
+            },
+          ])
+        },
+      })
```

Comment:
Once you've restored the menu stuff discussed [here](https://github.com/GMOD/Apollo3/pull/153#pullrequestreview-1166208849), you'll want to merge that branch in here and move this menu stuff to the same place.

Reply:
Issue150 is now merged